.circleci/config.yml

```diff
@@ -265,6 +284,19 @@ jobs:
       - store_artifacts:
           path: "test/integration/render-tests/index.html"
 
+  test-render-linux-firefox:
```

Comment:
Should we be able to test firefox on Mac, or expected on linux only? I ran it locally on Mac and these tests failed:  render-tests/custom-source/satellite
render-tests/debug/raster
render-tests/globe/globe-transition/tile-edges
render-tests/globe/globe-transition/pitch
render-tests/globe/globe-transition/bearing

Reply:
Also if it is expected that we can test render firefox locally - update integration docs?

Reply:
> Should we be able to test firefox on Mac, or expected on linux only?

I haven't specifically tested out that combination and see it out of scope of this PR, but I've tried to set up browser/OS configurations that we can scale to new combinations of browsers and OS's.
Of course ideally all browser/OS combinations should be supported and pass tests, but IMO it doesn't seem like a disaster to have a few tests fail locally, and preferable to reducing the test coverage on CI. We can look into those issues if and when we set up Ci for Firefox on Mac.

> Also if it is expected that we can test render firefox locally - update integration docs?

Yes, commands in `package.json` are the same everywhere, so we can now run firefox tests with `test-render-firefox` (`watch-render` already works with other browsers). Though I'm not sure what the use case is, is there a good reason for us to want to run this on dev machines?

Reply:
Nope just checking - By integration docs, I mean readmes. But not important if we are not planning to utilize this option locally and rely on circle ci runs for firefox!

Reply:
To clarify, I agree - don't think it necessarily to have this passing locally on mac - all good, was just checking that was what was expected. Thanks @SnailBones !